### PR TITLE
[AppStore Metadata] Update copy

### PR DESCRIPTION
**Pull request by alloy**
Updated name, keywords, and description all to be English as defined in #1272.

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by katarinabatina** [comment]
@alloy where can I take care of this in itunes connect?

**Comment by alloy** [comment]
@katarinabatina You don’t. Luckily people have built tools to automate this, which is what these text files are used for. So if you give the thumbs up on these changes (i.e. I did not misunderstand anything), I’m deploying them.

**Comment by katarinabatina** [comment]
woahhhh automation ~~ C O O L ~~ yup this looks great thanks @alloy!

**Comment by alloy** [comment]
Alrighty then!

Ignoring the danger fail, I guess I should have included #trivial or something.

File: fastlane/metadata/zh-Hans/keywords.txt [hunk 1]
```diff
@@ -1 +1 @@
-artsy, 艺术品、艺术家、博物馆、画廊、展览、购买艺术品、拍卖结果、艺术导览、艺术博览会、绘画、雕塑
\ No newline at end of file
+gallery,fairs,exhibition,collector,museum,auction,buy,artwork,artist,armory,bid,interior,banksy,fair
```
Comment (orta) [inline fastlane/metadata/zh-Hans/keywords.txt]: think changing the keywords might be a bad idea though

**Comment by katarinabatina** [comment]
@orta Jesse and I did research on why we'd move towards these

**Comment by alloy** [comment]
@orta Do you mean the semantic values of the keywords or replacing with unlocalised ones?

**Comment by orta** [comment]
Replacing

**Comment by orta** [comment]
But cool, just making sure it's not an accident

**Pull request merged**